includes/abstracts/abstract.llms.database.query.php

```diff
@@ -402,6 +405,28 @@ protected function setup_args() {
 
 	}
 
+	/**
+	 * Retrieve the prepared SQL for the SELECT clause.
+	 * Prepends `SQL_CALC_FOUND_ROWS` if `no_found_rows` arg is set to `false`.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $select_columns Optional. Columns to select. Default '*'.
+	 * @return string
+	 */
+	protected function sql_select_columns( $select_columns = '*' ) {
+
+		if ( ! $this->get( 'no_found_rows' ) ) {
+			$select_columns = 'SQL_CALC_FOUND_ROWS ' . $select_columns;
+		}
+
+		if ( $this->get( 'suppress_filters' ) ) {
+			return $select_columns;
+		}
+
+		return apply_filters( $this->get_filter( 'select_columns' ), $select_columns, $this );
```

Comment:
I thought this function was a good idea when I wrote it but years later I realize it makes it hard to read the code when trying to figure out what the filter actually does.

I'd like to remove it eventually.

Knowing this, while working on improving this and related classes if new filters need to be created I would like to not use this function. Please.

Reply:
Oh yeah I see, so:
```suggestion
		return apply_filters( "llms_{$this->id}_query_select_columns", $select_columns, $this );
```
I can change the other occurrences as well.

Reply:
Oh yeah I see, so:
```suggestion
		return apply_filters( "llms_{$this->id}_query_select_columns", $select_columns, $this );
```
I can change the other occurrences as well.